### DISCUSSION TITLE
Checkbox and Input Switch

### DISCUSSION THREAD

**Discussion by anchan42** [primefaces/community#1381] [PrimeVue] [answered]
How can I make the input switch work like the checkbox? i.e. tie the value up to an array?
The two components are quite similar but seem to have completely different underlaying API.

**Comment by tugcekucukoglu** [comment, marked as answer]
You could create your own component by wrapping the PrimeVue Checkbox. But I want to say they have different behaviors. 

References;
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio
